install.sh

```diff
@@ -16,8 +16,8 @@ function display_help()
     echo "       --fast                  Quick-build RCCL (local gpu arch only, no backtrace support)"
     echo "    -h|--help                  Prints this help message"
     echo "    -i|--install               Install RCCL library (see --prefix argument below)"
+    echo "    -l|--limit-nprocs          Limit the number of procs to 16 while building"
     echo "       --local_gpu_only        Only compile for local GPU architecture"
```

Comment:
Should be able to disable collective trace here with its own flag, not just only through --fast